presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -333,6 +333,50 @@ public void testSubstring()
         assertFunction("SUBSTRING('\uD801\uDC2Dend' FROM 2 FOR 3)", createVarcharType(4), "end");
     }
 
+    private static SqlVarbinary varbinary(String str)
```

Comment:
- Usually we put private helpers to the end of a file.
- Let's make the variable name complete: `string`